.travis.yml

```diff
@@ -0,0 +1,18 @@
+branches:
+  only:
+    - master
+cache:
+  directories:
+    - vendor
+sudo: required
+services:
+  - docker
+install:
+  - make bootstrap
+script:
+  - make test
+deploy:
+  provider: script
+  script: make ci-deploy
```

Comment:
Personal preference-- felt things were a little simpler by keeping all the deploy logic in one place.  (i.e. Deploy to local cluster when developing and CI deploy are both described by make targets.)  If there's a very compelling reason not to do this, I would be happy to reconsider that.